open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,749 @@
+                    y = (parseFloat($sessionElement.data('y')) || 0) + event.dy;
+
+                $sessionElement.css("-webkit-transform", "translate(" + x + "px, " + y + "px)");
+                $sessionElement.css("transform", "translate(" + x + "px, " + y + "px)");
+
+                $sessionElement.data('x', x);
+                $sessionElement.data('y', y);
+
+                $sessionElement.data("temp-top", roundOffToMultiple($sessionElement.offset().top - $(".tracks.x1").offset().top));
+
+                if (isSessionOverTimeline($sessionElement)) {
+                    updateSessionTimeOnTooltip($sessionElement);
+                } else {
+                    $sessionElement.tooltip("hide").attr("data-original-title", "");
+                }
+
+            },
+            // call this function on every dragend event
+            onend: function (event) {
+
+            }
+        });
+
+
+    interact(".session")
+        .resizable({
+            preserveAspectRatio: false,
+            enabled: true,
+            edges: {left: false, right: false, bottom: true, top: false}
+        })
+        .on("resizemove", function (event) {
+            if ($(event.target).hasClass("scheduled")) {
+                var target = event.target,
+                    x = (parseFloat(target.getAttribute("data-x")) || 0),
+                    y = (parseFloat(target.getAttribute("data-y")) || 0);
+
+                target.style.width = roundOffToMultiple(event.rect.width) + "px";
+                target.style.height = roundOffToMultiple(event.rect.height) + "px";
+
+                $(event.target).ellipsis();
+
+                updateSessionTimeOnTooltip($(event.target));
+            }
+        })
+        .on("resizeend", function (event) {
+            if ($(event.target).hasClass("scheduled")) {
+                var $sessionElement = $(event.target);
+                updateSessionTime($sessionElement);
+                $(document).trigger("scheduling:change", $sessionElement.data("session"));
+            }
+        });
+
+    interact(".track-inner").dropzone({
+        // only accept elements matching this CSS selector
+        accept: ".session",
+        // Require a 75% element overlap for a drop to be possible
+        overlap: 0.50,
+
+        ondropactivate: function (event) {
+            $(event.target).addClass("drop-active");
+        },
+        ondragenter: function (event) {
+            $(event.target).addClass("drop-now");
+        },
+        ondragleave: function (event) {
+            $(event.target).removeClass("drop-now");
+        },
+        ondrop: function (event) {
+            var $sessionElement = $(event.relatedTarget);
+            var $trackDropZone = $(event.target);
+
+            $trackDropZone.removeClass("drop-active").removeClass("drop-now");
+
+            addSessionToTimeline($sessionElement, {
+                track_id: parseInt($trackDropZone.parent().attr("data-track-id")),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should specify the radix of the mentioned string in parseInt calls.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2668227635/issues/source?bid=3233864&fileBranchId=3324174#l526)